### PR TITLE
Add private map quota

### PR DESCRIPTION
**Pull request by gonzaloriestra**
Related to https://github.com/CartoDB/cartodb-central/issues/2641

:warning: Do not merge until https://github.com/CartoDB/cartodb/pull/15405 is deployed

### CODE REVIEW

**Comment by gonzaloriestra** [comment]
retest this please

File: services/user-mover/export_user.rb [hunk 1]
```diff
@@ -159,7 +159,8 @@ class ExportJob
         }
       }.freeze
       # the next fields won't be discarded if set to NULL
-      TABLE_NULL_EXCEPTIONS = ['table_quota', 'public_map_quota', 'regular_api_key_quota', 'builder_enabled'].freeze
+      TABLE_NULL_EXCEPTIONS = ['table_quota', 'public_map_quota', 'public_map_quota', 'regular_api_key_quota',
```
Comment (alrocar) [inline services/user-mover/export_user.rb]:
```suggestion
      TABLE_NULL_EXCEPTIONS = ['table_quota', 'public_map_quota', 'private_map_quota', 'regular_api_key_quota', 
```

Comment (gonzaloriestra) [reply services/user-mover/export_user.rb]: Good catch!

**Comment by gonzaloriestra** [comment]
retest this please

**Pull request merged**